lib/Runtime/Base/JnDirectFields.h

```diff
@@ -657,12 +664,6 @@ ENTRY(__initializedNumberFormat)
 ENTRY(__initializedDateTimeFormat)
 ENTRY(localeWithoutSubtags)
 ENTRY(__matcher)
-ENTRY(__localeForCompare)
-ENTRY(__usage)
-ENTRY(__sensitivity)
-ENTRY(__ignorePunctuation)
-ENTRY(__caseFirst)
-ENTRY(__numeric)
```

Comment:
Were these unused (and now removed) or renamed? If renamed, why drop the __ prefix?

Reply:
Theyre not used in native code anymore, only in JS, so we didnt need to explicitly declare property IDs for them.